### ISSUE TITLE
pipboard: integrate facets for sample visualization

### ISSUE THREAD

**Issue by yorkie** [alibaba/pipcook#50]
https://pair-code.github.io/facets/ is for checking common problems in data/sample, so we could have a facets extension to achieve sample visualization.

**Comment by utkobe** [comment]
Exactly, now that we have standard (uniform) dataset format (csv, voc ...), we can just detect current dataset and show the visualization of data. I guess we can add another tab on top of pipboard and name it 'data visualization'

**Comment by utkobe** [comment]
Or we have the data tab right now to show the basic information of each dataset currently stored in pipcook-output log dir. Just add a 'visulize' button in each dataset and direct to visualization

**Comment by yorkie** [comment]
IMO, the facets extension will register the Dataset click event, when user clicks, we will open the facets page for that data :)

**Comment by yorkie** [comment]
This really depends on our Pipboard Extension APIs, so let's achieve that firstly :p
